server/lib/nl/detection/utils.py

```diff
@@ -96,6 +96,14 @@ def get_multi_sv_pair(
   return parts
 
 
+def get_top_sv_score(detection: Detection) -> float:
+  if is_multi_sv(detection):
```

Comment:
the return type is a float so shouldn't this return something? else, the return type hint could be a Union[float, None] or something like that?

Reply:
Sorry, not following the Q, it is returning the score, or 0?

Reply:
I'll merge now and handle something I missed based on your response

Reply:
Ooops sorry. Too late in the night. My screen was showing me a line break so it seemed like it was just a dangling return statement... my bad!